dev-libs/libappindicator/files/libappindicator-12.10.0-conditional-py-bindings.patch

```diff
@@ -0,0 +1,72 @@
+--- bindings/Makefile.am	2015-07-10 10:04:18.091199191 +0200
++++ bindings/Makefile.am	2015-07-10 10:19:47.047974439 +0200
+@@ -3,8 +3,10 @@
+ 	vala
+ else
+ SUBDIRS =		\
+-	python		\
+ 	vala
++if HAS_PYTHON
++SUBDIRS += python
++endif
+ endif
+ 
+ if HAS_MONO
+--- configure.ac	2015-07-16 13:28:41.007502838 +0200
++++ configure.ac_new	2015-07-16 13:53:18.118602877 +0200
+@@ -188,25 +188,37 @@
+ PYGTK_REQUIRED=2.14.0
+ PYGOBJECT_REQUIRED=0.22
+ 
+-AM_PATH_PYTHON(2.3.5)
+-AM_CHECK_PYTHON_HEADERS(,[AC_MSG_ERROR(could not find Python headers)])
++AC_ARG_ENABLE([python],
++	AC_HELP_STRING([--enable-python], [Enable python for GTK2 build]),,
++	[enable_python=yes])
+ 
+-PKG_CHECK_MODULES(APPINDICATOR_PYTHON,
++if test x"$with_gtk" = "x3" ; then
```

Comment:
Now I'm being picky but since I have other comments, you can fix this one as well. I'd suggest using consistent quoting on both sides, i.e. either `"x$with_gtk"` or `x"3"` :).